src/statefunctions/statecontroller.cc

```diff
@@ -40,7 +40,7 @@ void ExitGame(int game) {
 std::unique_ptr<GameState> InitGameState(const GameSettings& settings) {
   auto state = std::make_unique<GameState>();
   for (int i = 0; i < 4; i++) {
-    ControllerManager::Instance().NewController(settings.seatControllers.at(i));
+    state->players.at(i).controller = ControllerManager::Instance().NewController(settings.seatControllers.at(i));
```

Comment:
@aliceofhartley so I determined this was a fix to the segfault I was experiencing in my Api tests, NewController builds a pointer to the controller but then we weren't doing anything with it and needed to assign it to controller

Reply:
Lit